app/src/focusNightly/res/values/app.xml

```diff
@@ -2,6 +2,13 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<!DOCTYPE resources [ <!ENTITY app_name_entity "Firefox Focus Nightly"> ]>
```

Comment:
Why do we need this entity here when we have `app_name` below?

Reply:
Because it is used only in manifest android:label="@string/text_selection_search_action" and in XML from what I searched you can't do like in kotlin getString(R.string.text_selection_search_action, getString(R.string.app_name))